backend/src/controllers/teamController.ts

```diff
@@ -355,6 +376,76 @@ const addMembers = async (req: Request, res: Response) => {
   }
 };
 
+//Add Expense Account Data
+export const addExpenseAccountData = async (req: Request, res: Response) => {
+  const { id } = req.params
+  const callerAddress = (req as any).address
+  const expenseAccountData = req.body
+
+  try {
+    const team = await prisma.team.findUnique({
+      where: { id: Number(id) }
+    })
+    const ownerAddress = team?.ownerAddress
+    if (callerAddress !== ownerAddress) {
+      return errorResponse(403, `Action not authorized`, res)
```

Comment:
i think if u want to use 403 it should be `Action forbidden`,  if u want not authorized it should be 401

Reply:
Ah right, 403 is forbidden, thanks